pykern/pkio.py

```diff
@@ -336,6 +341,11 @@ def write_text(path, contents):
     from pykern import pkcompat
 
     fn = py_path(path)
```

Comment:
`write_binary()` uses `p = py_path(path)` but this uses `fn = py_path(path)`. A minor point but consistency is helpful

Reply:
Fix 54877fc